hngprojects/hng_boilerplate_expressjs
# Add /hng-test Page to Display Version, Backend URL, and API Response for Backend

## Description
This PR introduces a new endpoint `/probe` to the backend, which responds with a confirmation message. The front end will use this endpoint to verify the backend's availability and response.

## How should this be manually tested?
- Send a GET request to `/probe`.
- Verify the response is `I am the express api responding`.

## Checklist of what you did
- [x] Added the /probe endpoint:
- [x] The endpoint responds with the message `I am the express api responding`.

## Notes
The `/probe` endpoint is designed to confirm the backend's availability.

## Reference Issues
[FEAT] Add `/hng-test` Page to Display Version and Backend Info #358
